pkg/common/common.go

```diff
@@ -67,6 +67,8 @@ const (
 
 	// PvcApplyStorageProfileLabel tells whether the PVC should be rendered by the mutating webhook based on StorageProfiles
 	PvcApplyStorageProfileLabel = CDIComponentLabel + "/applyStorageProfile"
+	// PvcApplyStorageProfileMinimumSize tells whether the PVC mutating webhook rendering should apply only the StorageProfile minimum volume size if exists
+	PvcApplyStorageProfileMinimumSize = "minimumSize"
```

Comment:
Do we plan to add more `applyStorageProfile` modes? (accesmode, volume mode, etc)

Reply:
Even if it's just this one we should update the repository's docs with info about the possible options.

Reply:
This PR is just an option for consideration. I'm not sure it's the best one, and that's why it's on hold :)